### DISCUSSION TITLE
version 2.5 works where 1.x failed A LOT

### DISCUSSION THREAD

**Discussion by toughengineer** [maxonthegit/PPspliT#19] [General]
I tried version 2.5 on slides where version 1.27 (I believe) failed A LOT.
It worked the first time! I did not verify the correctness of splits, but this already relieves a huge pain where I had to delete animations and then manually split slides with a lot of animations.
Good job!
Thank you!

**Comment by maxonthegit** [comment]
Thank you very much for taking some time to report your improved experience!
